loadgen/config.example.yaml

```diff
@@ -0,0 +1,48 @@
+receivers:
+  loadgen:
+
+exporters:
+  otlp:
+    endpoint: "${env:ELASTIC_APM_SERVER_URL}"
+    headers:
+      Authorization: "ApiKey ${env:ELASTIC_APM_API_KEY}"
+#      Authorization: "Bearer ${env:ELASTIC_APM_SECRET_TOKEN}"
+    sending_queue:
+      enabled: false
+    timeout: 60s
+  debug:
+
+
+processors:
+  transform/rewrite:  # Rewrite telemetry to increase cardinality
+    trace_statements:
+      - context: span
+        statements:
+          # The worst way to generate a random ID, but is the simplest in OTTL
+          # Only randomize trace ID such that span relationships are still maintained
+          - set(trace_id.string, Substring(MD5(UUID()), 0, 32))
```

Comment:
As mentioned on Slack, we would ideally update trace IDs and span IDs so relationships are maintained. This can be done by adjusting each ID consistently across the batches within a file.

Not sure off hand how we would do that. One option could be to add support for OTTL to access client metadata, and then have the loadgen receiver inject a random value into client metadata. Alternatively it could add it as a scope attribute or something like that.

Reply:
After discussing with Felix, as span IDs are not guaranteed to be unique across different traces, just randomizing trace ID should cover most use cases. Let's leave randomizing span ID as a follow up.